cpu/src/csr.rs

```diff
@@ -0,0 +1,25 @@
+// Licensed under the Apache-2.0 license
+
+#[cfg(feature = "riscv")]
+pub fn mie_enable_external_interrupts() {
+    const MEIE: usize = 1 << 11;
+    unsafe {
+        core::arch::asm!("csrrs zero, mie, {r}", r = in(reg) MEIE);
+    }
+}
+
+#[cfg(feature = "riscv")]
```

Comment:
nit: IIUC, these mechanisms are veer-specific right? Should we gate on a `veer` feature flag rather than a generic `riscv` feature flag?